src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -66,18 +66,6 @@ internal abstract partial class AbstractNavigateToSearchService
             }
         }
 
-        private static string GetSearchName(DeclaredSymbolInfo declaredSymbolInfo)
-        {
-            if (declaredSymbolInfo.Kind == DeclaredSymbolInfoKind.Indexer && declaredSymbolInfo.Name == WellKnownMemberNames.Indexer)
-            {
-                return "this";
```

Comment:
this was 'undoing' how our index was storing data.  We don't need this anymore now that the index just stores the reasonable name for the indexer.